docs/runtime-contract.md

```diff
@@ -484,16 +484,16 @@ From the OCI spec:
 > article in the kernel documentation has more information about mount
 > propagation.
 
-This option should only be set by the operator or platform provider, and MUST
-NOT be configurable by the developer. As mount propagation may be part of the
-platform security hardening, operators MAY tune this over time as the threat
+This option SHOULD only be set by the operator or platform provider, and MUST
+NOT be configurable by the developer. As mount propagation might be part of the
+platform security hardening, operators might tune this over time as the threat
 environment changes.
 
 ### Masked Paths
 
-This option MAY only be set by the operator or platform provider, and MUST NOT
-be configurable by the developer. As masked paths may be part of the platform
-security hardening, operators may tune this from time to time as the threat
+This option SHOULD only be set by the operator or platform provider, and MUST NOT
+be configurable by the developer. As masked paths might be part of the platform
+security hardening, operators might tune this from time to time as the threat
```

Comment:
Ditto.

Reply:
Came concern as above - is this guidance/explanatory text or constraints?

Reply:
added new text to align with the other 2 things like this. SWYT